app/views/users/show.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1> Hello views </h1>
-<p> this is a test</p>
\ No newline at end of file
+<div class="user">
+<%= render 'partials/user', user: @user %>
+<div class='bio'>
+<p><%= @user.bio %></p>
+</div>
+<ul class='card'>
+    <% @user.recent_three_posts.each do |post| %>
+      <a href="/users/<%= @user.id %>/posts/<%= post.id %> ">
+        <%= render 'partials/post', post: post %>
```

Comment:
Wireframe shows that `comments counter` and `likes counter` are on right side in every post. KIndly compare your page layout with wireframe and design the page as it's shown.
![comments](https://user-images.githubusercontent.com/35504149/205401443-78b5db0e-0f9a-4066-936a-03688489a5a0.png)